homeassistant/components/rachio/switch.py

```diff
@@ -216,7 +217,12 @@ def entity_picture(self):
     @property
     def state_attributes(self) -> dict:
```

Comment:
Platform entities should only implement `device_state_attributes`, not `state_attributes`.

https://developers.home-assistant.io/docs/entity_index#generic-properties

Reply:
Will fix in a followup once other submitters get their changes in so I don't create any additional conflict pain for them